### PR TITLE
Add ScottoErgo Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
Add ScottoErgo handwired keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scottoalp/keymaps/scotto/config.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define options
+#define IGNORE_MOD_TAP_INTERRUPT
```
Comment (drashna) [inline keyboards/handwired/scottokeebs/scottoalp/keymaps/scotto/config.h:21]:
```suggestion
```

File: keyboards/handwired/scottokeebs/scottoergo/keymaps/scotto/config.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define options
+#define IGNORE_MOD_TAP_INTERRUPT
```
Comment (drashna) [inline keyboards/handwired/scottokeebs/scottoergo/keymaps/scotto/config.h:21]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scottoergo/info.json [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT_ortho_3x10_6": {
```
Comment (dunk2k) [inline keyboards/handwired/scottokeebs/scottoergo/info.json:39]: Entirely optional, but changing `LAYOUT_ortho_3x10_6` to `LAYOUT_split_3x5_3` will allow community layouts to automatically be assigned (namely miryoku).  
Would have to apply this change to all `keymap.c` files also.

Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scottoergo/info.json]: Changed in [d67b78c](https://github.com/qmk/qmk_firmware/pull/20944/commits/d67b78c9425552a43f8df44d87a577469d26a0e0).

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/scottokeebs/scottoergo/readme.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+# Compiling
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/jscotto/scottoergo:default
+
+Flashing example for this keyboard:
+
+    make handwired/jscotto/scottoergo:default
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoergo/readme.md:19]:
```suggestion
    make handwired/scottokeebs/scottoergo:default

Flashing example for this keyboard:

    make handwired/scottokeebs/scottoergo:default
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scottoergo/readme.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# ScottoErgo
+
+![ScottoErgo](https://i.imgur.com/FtdSJRBh.jpg)
+
+A 36 key column staggered ergonomic keyboard with 10 degrees of tenting and 20 degrees of angle. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: RP2040
+-   Hardware Availability: [Amazon](https://amazon.com)
```
Comment (fauxpark) [inline keyboards/handwired/scottokeebs/scottoergo/readme.md:9]:
```suggestion
* Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
* Hardware Supported: RP2040
* Hardware Availability: [Amazon](https://amazon.com)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**